network/sites.csv

```diff
@@ -2237,7 +2237,8 @@ WI14,10,-37.52216,177.18849,25,,WGS84,External GPS Device,2011-06-13T00:00:02Z,2
 WI15,10,-37.52165,177.18668,33,,WGS84,External GPS Device,2011-06-13T00:00:02Z,2011-11-23T00:00:00Z
 WI227,02,-37.52188,177.189942,9,,WGS84,External GPS Device,2019-04-10T01:40:00Z,9999-01-01T00:00:00Z
 WI227,03,-37.52188,177.189942,9,,WGS84,External GPS Device,2019-06-17T23:40:00Z,9999-01-01T00:00:00Z
-WIAZ,10,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2010-07-27T03:15:00Z,9999-01-01T00:00:00Z
+WIAZ,10,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2010-07-27T03:15:00Z,2024-10-09T21:58:00Z
+WIAZ,11,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2024-10-09T21:59:00Z,9999-01-01T00:00:00Z
```

Comment:
do we need to change the coordinates a bit (like increase the decimal precision 😄 ) just to distinguish between `LC 10` and `LC 11`? but not to worry, i'm ok with this. 👍

Reply:
I'll update channel naming for LC11 now.

Reply:
@Thomas-Benson @mnaguit 

channel names updated `Sun Oct 13 20:27:19 UTC 2024`